lib/portlayer/exec/container_cache.go

```diff
@@ -79,3 +117,109 @@ func (conCache *containerCache) Remove(idOrRef string) {
 	}
 
 }
+
+func isContainerID(id string) bool {
```

Comment:
we have a single map[string]*Container that has both IDs and Managed Object References as the key -- this allows us to distinguish between the two